charts/stable/otel-collector/values.yaml

```diff
@@ -0,0 +1,179 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: otel/opentelemetry-collector
+  # -- image tag
+  tag: 0.46.0
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- command is set to `otelcol` when using otel/opentelemetry-collector image but should be
+# changed to `otelcontribcol` if the image is changed to otel/opentelemetry-collector-contrib.
+# @default -- otelcol
+command:
+- "/otelcol"
```

Comment:
This logic could be handled by the `hardcodedValues` solution as proposed in my comment on `common.yaml`

Reply:
Which is preferable?

1. Assume everyone is okay with running the 50mb image with all the contrib plugins and just set the command to `/otelcontribcol` and the image to `otel-collector-contrib`. 
2. Expose a value that hints people into how to enable the contrib otel collector release

I think principle of least surprise is probably hard-coding the contrib release.